src/Controller/ListPageRssController.php

```diff
@@ -164,12 +191,68 @@ public function build(NodeInterface $node): CacheableResponse {
    *
    * @param \Drupal\node\NodeInterface $node
    *   The node containing the list.
+   * @param \Drupal\Core\Cache\CacheableMetadata $cache_metadata
+   *   The current cache metadata.
    *
    * @return array
    *   Array of items to be rendered.
    */
-  protected function getItemList(NodeInterface $node): array {
-    return [];
+  protected function getItemList(NodeInterface $node, CacheableMetadata $cache_metadata): array {
+    $configuration = ListPageConfiguration::fromEntity($node);
+    // Always sort by the updated date descending.
+    $configuration->setSort([
+      'name' => 'changed',
+      'direction' => 'DESC',
+    ]);
+    $execution_result = $this->listExecutionManager->executeList($configuration);
```

Comment:
There is requirement stating that limit for rss pages should be 25 items, we need to set it here and then test it. Currently default limit 10 is used.

Reply:
@imanoleguskiza Indeed, you can pass the limit to the configuration object.